neusomatic/CMakeLists.txt

```diff
@@ -14,7 +14,6 @@ else()
 endif()
 
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/bin)
-set( CMAKE_CXX_FLAGS " ${CMAKE_CXX_FLAGS} -fopenmp " )
 set( CMAKE_CXX_FLAGS " ${CMAKE_CXX_FLAGS} -Wall -Wno-deprecated -Wno-unused-variable " )
```

Comment:
@roger-liu-bina why did you remove OpenMP? What was it being used for originally?

Reply:
@marghoob originally we have the condensed_array loading from the a MSA, that part can use multiple threads. Now we initialize the `condensed_array` on the fly, and now it is a single thread job.